etna/packages/etna-js/components/inputs/dropdown_autocomplete_wrapper.jsx

```diff
@@ -0,0 +1,25 @@
+// Wraps the DropdownAutocomplete component so it will work with the
+// list_input to act as a multi-select.
+import React from 'react';
+
+import DropdownAutocomplete from './dropdown_autocomplete';
+
+export default function DropdownAutocompleteInput({
+  onChange,
+  list,
+  onBlur,
+  ...otherProps
+}) {
+  function onChangeWrapper(value) {
```

Comment:
Minor nit -- you can use `const onChangeWrapper = useCallback((value) => { onChange(value); onBlur(); })` to slightly improve performance and make the onChangeWrapper reference stable.